cmd/root.go

```diff
@@ -62,6 +62,8 @@ func init() {
 	RootCmd.PersistentFlags().String("network", "", "The pegnet network to target. <MainNet|TestNet>")
 	RootCmd.PersistentFlags().Bool("testing", false, "Sets all activation heights to 0 so you can run on a local net")
 
+	RootCmd.PersistentFlags().Int64Var(&common.FloatingPegPriceActivation, "v3", 0, "Sets the v3 activation height")
```

Comment:
i think this will always override whatever we end up hardcoding. make this behave like the other flags and only override common.FPPA when set

Reply:
I'm just going to drop it now. I was going to drop it later, as it was just for QA.